spec/integration/aws_vpc_spec.rb

```diff
@@ -0,0 +1,331 @@
+require 'spec_helper'
```

Comment:
I think the pattern looks good!  At some point we should allow it to talk to a real AWS as well as a fake one, though :)